memory/main.py

```diff
@@ -1938,81 +1927,6 @@ def party_format_cursor_2():
     return ret_val
 
 
-def get_party_format_from_text(front_line):
-    logger.debug(f"||| FRONT LINE VARIABLE: {front_line}")
-    if front_line == "kimahri":
-        order_final = [0, 3, 2, 6, 4, 5, 1]
```

Comment:
Do we need to worry about the non in party member locations?

Reply:
Nope. Because of the TAS inputs, it's better to just let the back party be however it is cuz it's faster to scroll to where it is when needed then to move it into a better spot for later, and the function automatically does the backfill.